D10 - Logging.md

```diff
@@ -1,5 +1,10 @@
 # D10 - Logging
 
+In the case of applications involving a dockerized environment, the logging process involves JSON-formatted logs. A logging driver manages the logs emitted by the docker through its stdout and stderr streams. The logging driver is generally located at the local disk of the respective Docker host. There can often be latency differences in the case of different drivers. To prevent this, the log management setup can be made with a JSON-file driver backed up with the docker blocking mode. One of the challenges in the case of containers is the multi-tiered nature of containers. Hence the defined infrastructure must segregate various log events into individual processes.
+
+Various logging strategies can be followed in the case of containerized applications. These include but are not limited to logging via a dedicated logging container, using data volumes, working through the sidecar approach, or logging through an application. One can simply carry on the workflow using the docker logging driver as well, however, each mentioned approach has benefits of its own.
+
```

Comment:
Should I explain the types?

Reply:
I recommend to start a migration to "container" to be more general, but this is a bigger topic I assume.
I would high lite that containers should log on stdout/stderr streams like this:

> In the case of components in container environments should log through _stdout_ and/or _stderr_. Ideally in JSON format.

I wouldn't push to hard the logging manager because there are multiple options on gathering and processing logs like mentioned in the second paragraph.